docs/extensions/designing-extensions.md

```diff
@@ -18,45 +18,43 @@ has_children: false
   - [Widget Extension Guidelines](#widget-extension-guidelines)
 
 ## Introduction
-If you are just getting started with Extensions, use the recommendations in this guide to put your best foot forward for LumApps users. If you already shipped an extension, use this guide to take your extension to the next level.
+If you are just getting started with extensions, use the following recommendations to bring the best for LumApps users. 
+
+If you already shipped an extension, use this guide to take your extension to the next level.
 
 ## General guidelines
-Extensions provide ways to extends LumApps, so we provide guidelines to help ensure your experiences are harmonious with the overall LumApps experience. 
+Extensions provide ways to extend LumApps. We provide guidelines to help ensure your experiences are harmonious with the overall LumApps experience. 
 
-The guidelines in this section apply to all extension types. Guidelines for specific extension types appear in Backend Extension Guidelines, Share To Extension Guidelines, and Widget Extension Guidelines.
+The guidelines in this section apply to all extension types. Guidelines for specific extension types appear in Share To extension guidelines, and Widget extension guidelines.
 
 
 ### Terminology
```

Comment:
Don't know if it just in my Vscode, but this title seems not to be interpreted as markdown (had to replace the space between ### and Terminology)